chainweb.openapi.yaml

```diff
@@ -2556,7 +2700,7 @@ paths:
                       The chain selection made by the Node. This is
                       informational. Generally, miner should not care about the
                       chain.
-                    type: byte
+                    type: integer
```

Comment:
I think, this (and the values below) are actually binary encoded as bytes. Is this a 3.0 vs. 3.1 problem again? Or is `byte` generally not supported?

Reply:
`byte` isn't in the spec as a type, hence even parsing this document yields an error with `type: byte`. 

The schema isn't checked at all anyway, because it's a JSON schema, and this payload is not JSON. It's purely informational.

Reply:
The content of the response is just bytes (`application/octet-stream`). The property description is just an "informational" description of the byte content.

I think the correct specification according to https://swagger.io/specification/#data-types would be


```yaml
type: string
format: binary
```

same below